tests/e2e/specs/admin/anchor-linking.js

```diff
@@ -50,7 +50,7 @@ describe('AMP developer tools settings', () => {
 			'page=amp-options',
 			'other-settings'
 		);
-	});
+	}, 400000);
```

Comment:
Is this 400 seconds?

Reply:
Yes, by default we are using 300 seconds.